dist/rpm/os-autoinst.spec

```diff
@@ -107,15 +105,12 @@ for i in 07-commands 13-osutils 14-isotovideo 18-qemu-options 18-backend-qemu 99
 done
 
 %build
-mkdir -p m4
-autoreconf -f -i
-%configure --docdir=%{_docdir}/%{name}
-make INSTALLDIRS=vendor %{?_smp_mflags}
+%define __builder ninja
+%cmake -DOS_AUTOINST_DOC_DIR:STRING=%{_docdir}/%{name}
+%cmake_build
 
 %install
-%make_install INSTALLDIRS=vendor
-# only internal stuff
-rm -r %{buildroot}/usr/lib/os-autoinst/tools/
+%cmake_install install-openvswitch
```

Comment:
Can you explain that in an in-file comment why not just `%cmake_install`?

Reply:
Shouldn't it be clear that we want to install additional, openvswitch-related files here? (I've excluded these files from the regular install target because they are apparently not considered generally useful. At least I suppose so from the fact that we also ship them in the separate `os-autoinst-openvswitch` package.)

Reply:
I don't understand. We did not need to explicitly need to mention this special target in before. What happens if we just do `%cmake_install`?

Reply:
That macro adds the target `install` to the make/ninja invocation. That target will install everything which is so far contained by the `os-autoinst` package. The target `install-openvswitch` will only install everything which is so far contained by the `os-autoinst-openvswitch` package. Since we're creating both packages here at once I'm using `install-openvswitch` here (in addition to `install` which comes from the macro).